coopersystem-fsd/pcc
# Add course classes block

## Description

- Participants will be displayed as Trainers or Instructors;
- Added block to show course classes;
- Moved payment block to bottom of event page.

## Steps to test

1. Create a Course type event;
2. Fill in the class fields

**Expected behavior:** The topics of the classes will be displayed in the content of the event of type Course

## Additional information
